docs/view_components/menu_bar.rst

```diff
@@ -1,3 +1,5 @@
+.. _menu-bar-label:
```

Comment:
This should be placed in the `tool_interface.py` file according to the template :)